pycbc/types/frequencyseries.py

```diff
@@ -561,6 +561,13 @@ def match(self, other, psd=None,
                      low_frequency_cutoff=low_frequency_cutoff,
                      high_frequency_cutoff=high_frequency_cutoff)
 
+    def plot(self, **kwds):
+        """ Basic plot of this frequency series
+        """
+        from matplotlib import pyplot
+        plot = pyplot.plot(self.sample_frequencies, self, **kwds)
```

Comment:
This isn't correct for a `frequencyseries` As `self` is complex, this will silently cast to `real` (or not so silently). What is best to plot here? Both `self.real() and self.image()` or `abs` and `phase`?

Reply:
(Unless the frequencyseries is a PSD ....)

Reply:
@spxiwh Right, I was mostly thinking about a PSD (I plot a lot of those). You are right though for complex types. Do you have a preference? 

One can of course with this function do the following as a side note.

```
hp.real().plot()
hc.real().pot()
abs(hc).plot()
numpy.angle(hc).plot()
```

The last one works because our array types is explicitly compatible with numpy so the return product of ufuncs on our types is our type back with the elements operated on by whatever the operation is.

Reply:
Maybe we could have specialized methods or kwargs, like so:
```python
hp.plot_mag()
hp.plot_phase()
```
or
```python
hp.plot(complex='mag')
hp.plot(complex='phase')
hp.plot(complex='real')
```
etc.

Reply:
@titodalcanton kwarg seems fine to me, the question though is what the default should be? I'm leaning towards the default is still as now, but I can swayed from that.

Reply:
Here is a possible behavior:
* Real timeseries -> no kwarg needed, just plot it.
* Complex timeseries, kwarg not specified -> either raise a ValueError, or assume mag (and maybe print a warning).
* Complex timeseries, kwarg specified -> do what the kwarg says.

Reply:
My preference:

Real timeseries -> no kwarg needed, just plot it.
Complex timeseries, kwarg not specified -> Plot the real *and* complex components 
Complex timeseries, kwarg specified -> do what the kwarg says.

Reply:
I'll go with the suggestion from @spxiwh

Reply:
@spxiwh @titodalcanton I've updated the code to just plot both the real and imaginary in this case. There is not extra keyword argument at this time, but please feel free to add that at your discretion in a future PR. I'm happy if we can add features that will make things easier for people.